### PR TITLE
add docment for 99_pytorch_doc

### PR DESCRIPTION
**Pull request by feynlee**
This fixes issue #55.

### CODE REVIEW

**Comment by muellerzr** [comment]
@jph00 could you please review this? It's not the typing library, and honestly it does make sense somewhat. But would like to know your thoughts. Thanks! 😄

File: fastai/_pytorch_doc.py [hunk 1]
```diff
@@ -2,11 +2,17 @@
...
+from types import ModuleType
+
 # Cell
 PYTORCH_URL = 'https://pytorch.org/docs/stable/'
 
 # Cell
-def _mod2page(mod):
+def _mod2page(
+    mod:ModuleType, # A pytorch module
+) -> str :
```
Comment (tmabraham) [inline fastai/_pytorch_doc.py:14]:
```suggestion
) -> str:
```

File: fastai/_pytorch_doc.py [hunk 2]
```diff
@@ -2,11 +2,17 @@
...
+# Cell
+from types import ModuleType
+
 # Cell
 PYTORCH_URL = 'https://pytorch.org/docs/stable/'
 
 # Cell
-def _mod2page(mod):
+def _mod2page(
+    mod:ModuleType, # A pytorch module
```
Comment (tmabraham) [inline fastai/_pytorch_doc.py:13]:
```suggestion
    mod:ModuleType, # A PyTorch module
```

File: fastai/_pytorch_doc.py [hunk 3]
```diff
@@ -2,11 +2,17 @@
...
+) -> str :
+    "Get the web page name for a pytorch module"
```
Comment (tmabraham) [inline fastai/_pytorch_doc.py:15]:
```suggestion
    "Get the webpage name for a PyTorch module"
```

File: fastai/_pytorch_doc.py [hunk 4]
```diff
@@ -17,7 +23,10 @@ def _mod2page(mod):
 import importlib
 
 # Cell
-def pytorch_doc_link(name):
+def pytorch_doc_link(
+    name:str, # Name of a pytorch module, class or function
```
Comment (tmabraham) [inline fastai/_pytorch_doc.py:27]:
```suggestion
    name:str, # Name of a PyTorch module, class or function
```

**Review by tmabraham — CHANGES_REQUESTED** [review]

File: fastai/_pytorch_doc.py [hunk 5]
```diff
@@ -17,7 +23,10 @@ def _mod2page(mod):
 import importlib
 
 # Cell
-def pytorch_doc_link(name):
+def pytorch_doc_link(
+    name:str, # Name of a PyTorch module, class or function
```
Comment (tmabraham) [inline fastai/_pytorch_doc.py:27]:
```suggestion
    name:str # Name of a PyTorch module, class or function
```

**Review by tmabraham — COMMENTED** [review]
Just one very minor change, otherwise I'm good.

File: fastai/_pytorch_doc.py (hunk 5 above)
Comment (feynlee) [reply fastai/_pytorch_doc.py]: updated.

**Review by tmabraham — APPROVED** [review]

**Review by warner-benjamin — APPROVED** [review]
LGTM

**Pull request merged**